### DISCUSSION TITLE
Citation

### DISCUSSION THREAD

**Discussion by Jiaze-Sun** [rsagroup/rsatoolbox#284] [Q&A]
Dear developers,

Could you let me know how I cite this toolbox properly? 
I didn't find information from the website. The resource information for this toolbox on RRID portal remains with the matlab version.
Thank you!

**Comment by JasperVanDenBosch** [comment]
Thanks for asking. We are working on a manuscript on the toolbox, but for now it's ok to just include a link to this GitHub repository.

**Reply by Jiaze-Sun** [reply]
Thanks for replying.
